linkml/generators/markdowngen.py

```diff
@@ -296,7 +296,15 @@ def element_header(self, obj: Element, name: str, curie: str, uri: str) -> None:
             obj_type = 'Subset'
         else:
             obj_type = 'Class'
-        self.header(1, f"{obj_type}: {name}" + (f" _(deprecated)_" if obj.deprecated else ""))
+
+        # TODO: for some reason obj.deprecated dosn't appear to be True even
+        # ...though 'deprecated' in obj.slots. When that's fixed, can update
+        # ...this clause to be simpler. - joeflack4 2021/09/23
+        if (hasattr(obj, 'slots') and 'deprecated' in obj.slots) or obj.deprecated:
+            self.header(1, f"{obj_type}: ~~{name}~~ _(deprecated)_")
```

Comment:
Since we have strikethrough now, should we get rid of `_(deprecated)_`, or keep it? I'm kind of agnostic; it might help for extra clarity.
<img width="751" alt="Screen Shot 2021-09-23 at 8 03 14 PM" src="https://user-images.githubusercontent.com/13045020/134599091-8cd476db-3a4d-46ee-b8cb-616b9e381041.png">

Reply:
I say keep it